dataprofiler/profilers/graph_profiler.py

```diff
@@ -23,7 +25,6 @@ def __init__(self, data, options=None):
         Initialize Graph Profiler.
 
         :param data: data
```

Comment:
```suggestion
        :param data: data
        :type name: String
```

Reply:
I think this should be: one of these two types:
`data_readers.graph_data.GraphData or nx.Graph`